src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java

```diff
@@ -1011,4 +1011,19 @@ private boolean isVariableMatchesIgnorePattern(String variable) {
         return matcher.matches();
     }
 
+    /**
+     * Check if the statement is zero-distance token.
+     * Modifiers, types and resources are indicators of function or class or variable definitions.
```

Comment:
This is not required, as it is clearly visible from code.
Remove it.
We need in comment something that code might not be conveying to reader.

Reply:
Done.